packages/ffe-form/less/radio-block.less

```diff
@@ -46,53 +46,75 @@
         cursor: pointer;
         transition: border-color @ffe-transition-duration @ffe-ease;
 
+        .native & {
+            @media (prefers-color-scheme: dark) {
+                border: 2px solid @ffe-grey-darkmode;
+                color: @ffe-white-darkmode;
+                background: @ffe-grey-charcoal-darkmode;
+            }
+        }
+
         &::before {
             .ffe-sb1-radioblob();
 
             left: 20px;
             top: 10px;
             z-index: 1;
-        }
-    }
 
-    &__header {
-        display: block;
-        padding: 8px 60px 7px;
-        cursor: pointer;
-        transition: background-color @ffe-transition-duration @ffe-ease,
-            border-color @ffe-transition-duration @ffe-ease;
-    }
+            .native & {
+                /* stylelint-disable max-nesting-depth */
```

Comment:
Litt usikker på om jeg burde flytte litt nesting ut, eller om jeg bare kan ignorere regelen her

Reply:
Det er mulig vi må revurdere lint-reglene når dark mode innføres, gitt at en enkel overskriving av en property for å støtte dark mode bruker opp to nivåer med nesting. Jeg synes det er mest ryddig at vi overskriver stiler for dark mode i kodelinjene som umiddelbart etterføler den opprinelige deklarasjonen, så langt det lar seg gjøre.